docs/Reference/API-Methods.md

```diff
@@ -4147,6 +4147,43 @@ Distributes a signed, RLP encoded [private transaction](../HowTo/Send-Transactio
     } 
     ```
 
+### priv_getEeaTransactionCount
+
+Returns the private transaction count for the specified account and [privacy group](../../Concepts/Privacy/Privacy-Groups/#eea-compliant-privacy).
+
+!!! important 
+    If sending more than 1 transaction to be mined in the same block (that is, you're not waiting for 
+    the transaction receipt), you must calculate the private transaction nonce outside Besu. 
+
+**Parameters** 
+
+`data` - Account address
+
+`data` - Orion address of the sender
```

Comment:
Do you want to add that these are base64 encoded. The sender address, as well as the recipient addresses

Reply:
We probably should

Reply:
Done.